CODIGO/frmConnect.frm

```diff
@@ -742,27 +743,55 @@ Private Sub render_MouseUp(Button As Integer, Shift As Integer, x As Single, y A
                 End If
 
             End If
-       
-            If x > 297 And x < 314 And y > 321 And y < 340 Then 'ok
-    
-                If frmCrearPersonaje.lstHogar.ListIndex < frmCrearPersonaje.lstHogar.ListCount - 1 Then
-                    frmCrearPersonaje.lstHogar.ListIndex = frmCrearPersonaje.lstHogar.ListIndex + 1
-                Else
-                    frmCrearPersonaje.lstHogar.ListIndex = frmCrearPersonaje.lstHogar.ListIndex - 1
-
-                End If
+            
+            ' Hogar
+            ' Shugar: Arreglo los botones para seleccionar el hogar inicial.
+            ' También limito la selección del hogar a Ulla, Nix, Arghal y Forgat.
+            ' El frmCrearPersonaje.lstHogar.ListIndex arranca en 0: Ullathorpe
+            
+            
+            If x > 416 And x < 433 And y > 323 And y < 338 Then
+            
+                ' Botón de la derecha: es el que aumenta el index.
+                ' Si llega al índice máximo (que es el Count-1), vuelve a cero como un buffer circular
+                ' Uso Select Case para saltearme algunas ciudades, me quedo solo con:
+                ' 0: Ulla, 1: Nix, 4: Arghal, 5: Forgat
+                
+                Select Case frmCrearPersonaje.lstHogar.ListIndex
+                    Case 0
+                        frmCrearPersonaje.lstHogar.ListIndex = 1
```

Comment:
En vez de usar estos números hardcodeados habría que utilizar el Enum de ciudades 

También habría que crear un sub método para evitar repetir este Select Case en 2 lados con la misma lógica

Reply:
Listo @RecoX, usé el enum de ciudades y armé dos Sub, uno para el botón que avanza y el otro para el que retrocede. Si notás que las sub tienen un -1 es porque el enum arranca en 1 y necesito que ahí arranque en 0. No cambié la definición del enum por si las dudas.